users/serializers.py

```diff
@@ -0,0 +1,30 @@
+from rest_framework import serializers
+from django.contrib.auth import get_user_model
+
+
+UserModel = get_user_model()
+
+
+class UserSerializer(serializers.ModelSerializer):
+    password = serializers.CharField(
+        min_length=8,
+        max_length=50,
+        style={'input_type': 'password'},
+        write_only=True)
+    password2 = serializers.CharField(
+        min_length=8,
+        max_length=50,
+        style={'input_type': 'password'},
+        write_only=True)
+
+    class Meta:
+        model = UserModel
+        fields = ('username', 'password', 'password2', 'email')
+
+    def create(self, validated_data):
+        user_object = UserModel(
+            username=validated_data.get('username'),
+            email=validated_data.get('email'))
+        user_object.set_password(validated_data.get('password'))
```

Comment:
As password is required it's better to use:
validated_data['password']
the same for the email